FirebaseAuth/Tests/Sample/Sample/UserInfoViewController.m

```diff
@@ -108,4 +111,16 @@ - (NSString *)multiFactorString {
   return string;
 }
 
+- (NSString *)passkeysString {
+  NSMutableString *string = [NSMutableString string];
+
+  for (FIRPasskeyInfo *info in _user.enrolledPasskeys) {
+    [string appendString:info.name];
+    [string appendString:@" - "];
+    [string appendString:info.credentialID];
```

Comment:
Add separators between the passkey strings?

Reply:
Added a space following MFA